File: test/jskills/trueskill/TwoTeamTrueSkillCalculatorTest.java
package jskills.trueskill;

import org.testng.annotations.Test;

public class TwoTeamTrueSkillCalculatorTest
{
    @Test
    public void TwoTeamTrueSkillCalculatorTests()
    {
        TwoTeamTrueSkillCalculator calculator = new TwoTeamTrueSkillCalculator();

        // This calculator supports up to two teams with many players each
        TrueSkillCalculatorTests.TestAllTwoPlayerScenarios(calculator);
        TrueSkillCalculatorTests.TestAllTwoTeamScenarios(calculator);
    }
}

File: TrueSkillEngine/src/test/scala/jskills/trueskill/TwoTeamTrueSkillCalculatorTest.scala
package jskills.trueskill

import org.junit.Before
import org.junit.Test

class TwoTeamTrueSkillCalculatorTest extends TrueSkillCalculatorTestUtils {

  var calculator: TwoTeamTrueSkillCalculator = null

  @Before def setup() {
    calculator = new TwoTeamTrueSkillCalculator()
  }

  @Test def testAllTwoPlayerScenarios() {
    // This calculator supports up to two teams with many players each
    TestAllTwoPlayerScenarios(calculator)
  }

  @Test def testAllTwoTeamScenarios() {
    // This calculator supports up to two teams with many players each
    TestAllTwoTeamScenarios(calculator)
  }
}